v5/pivotal/stories.go

```diff
@@ -116,6 +116,24 @@ type Comment struct {
 	UpdatedAt           *time.Time `json:"updated_at,omitempty"`
 }
 
+type Blocker struct {
+	Id          int        `json:"id,omitempty"`
+	StoryId     int        `json:"story_id,omitempty"`
+	PersonId    int        `json:"person_id,omitempty"`
+	Description string     `json:"description,omitempty"`
+	Resolved    bool       `json:"resolved,omitempty"`
+	CreatedAt   *time.Time `json:"created_at,omitempty"`
+	UpdatedAt   *time.Time `json:"updated_at,omitempty"`
+}
+
+type BlockerRequest struct {
+	ProjectId   int    `json:"project_id,omitempty"`
+	StoryId     int    `json:"story_id,omitempty"`
+	PersonId    int    `json:"person_id,omitempty"`
+	Description string `json:"description,omitempty"`
+	Resolved    bool   `json:"resolved,omitempty"`
+}
```

Comment:
This struct should be

```golang
type BlockerRequest struct {
    Description string `json:"description,omitempty"`
    Resolved    *bool  `json:"resolved,omitempty"`
}
```

Would you mind adding `StoryService.UpdateBlocker` as well? You can basically copy `UpdateStory` and change it.